ci/travis/osx/before_install.sh

```diff
@@ -18,7 +18,8 @@ brew remove gdal || true
 
 brew tap osgeo/osgeo4mac
 brew update
-brew install osgeo/osgeo4mac/qgis-214 --without-postgresql --only-dependencies
+brew install grass-70
```

Comment:
Ideally, it should be `brew install osgeo/osgeo4mac/grass-70`, though this will work.